app/features/onboarding/constants.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import {
+    ConferenceURLSpotlight,
+    SettingsDrawerSpotlight,
+    NameSettingSpotlight,
+    EmailSettingSpotlight,
+    ServerSettingSpotlight
+} from './components';
+
+export const onboardingSteps = [
+    'conference-url',
+    'settings-drawer-button',
+    'name-setting',
+    'email-setting',
+    'server-setting'
+];
+
+export const onboardingComponents = {
```

Comment:
As we have to persist which onboarding components have been shown already, we can't directly use components. Hence we have to map them with strings.